src/views/RecIM/components/List/Listing/Listing.module.scss

```diff
@@ -24,22 +25,31 @@
   padding: 0;
 }
 
+.redButton {
+  background-color: $secondary-red;
+  color: $neutral-white;
+}
+
+.redButton:hover {
+  background-color: $alternative-red;
+}
+
 .rejectButton {
-  background-color: #900000;
+  background-color: $primary-blue;
   color: $neutral-white;
 }
 
 .rejectButton:hover {
-  background-color: #b00000;
+  background-color: $primary-blue-tint;
 }
 
 .acceptButton {
-  background-color: #007000;
+  background-color: $primary-cyan;
   color: $neutral-white;
 }
 
 .acceptButton:hover {
-  background-color: #009000;
+  background-color: $primary-cyan-shade;
 }
```

Comment:
You shouldn't need all these button styles, you can just the built-in MUI color props. `reject` and `accept` can definitely be replaced with `color="primary"` and `color="secondary"` respectively. I can't remember if we have the red color setup as a button color yet, but if not you can easily add it.

Reply:
Is this true? The MUI IconButton takes `color` as a parameter (true enough) but this changes the `color` property as expected which is in this case would have been the little `✔️` or `✖️` inside of the circular icon. Whereas I want to modify the outside circular button and maintain the white   `✔️` or `✖️` .

(@cpabbot commented, I forgot to submit this comment)

Reply:
Thanks @cpabbot and @amos-cha for clarifying - I didn't realize this was being used only for Icon Buttons. It's true that MUI doesn't currently support outlined Icon Buttons - that is a limitation of Material Design v2. It's been fixed by Material Design v3, but MUI is not being quick to adopt it (for a number of reasons that we don't need to get into here).

For now, I think what you have is a fine way to achieve a contained icon button. Maybe you could consider renaming the classes to clarify that they're for icon buttons only?

Reply:
renamed them to specify icon: i.e `acceptIcon`